string.go

```diff
@@ -28,16 +28,16 @@ func RandomAlphaString(size int) string {
 		if err != nil {
 			panic(err)
 		}
-		val := int(valBig.Int64())
+		val := valBig.Int64()
```

Comment:
I appreciate the very local fix. I'm not sure I grok what a 32-bit (arm?) build has to do with int(bigInt.Int64()) -> a negative number. I'm curious if we know whether we're:
* Working around a golang bug or
* We mis-read/neglected reading the golang documentation describing this behavior of `big.Int`

So as a devil's advocate to the fix here, I feel the complexity of maintaining this is the use of a cryptographically secure number generator. Which goes through this bigint stuff.

I don't think we need cryptographic secure number generation here. There's no (documented) production use of the function in rdk/app/goutils. And if we do need cryptographic generation, we shouldn't expose some string-ifying function for it and let users dictate how long the output should be.

Can we instead just do:
```
alphas = fmt.Sprintf("%s%s", alphaLowers, strings.ToUpper(alphaLowers))

...
idx := math/rand.Intn(len(alphas))
val := alphas[idx]
```
?

Reply:
int(random_positive_64_bit_number) on a 32-bit platform can be negative from truncation I think

Reply:
Oh I see now. The reproducer is much more clear about what's going on than this code. If my above suggestion is too much code slinging for a "just fix it" issue can we just change:
```
	maxIntBig       = big.NewInt(math.MaxInt64)
```
to `MaxInt32` instead?

The 64-bit numbers here are just to make the selection "as uniform as possible". 32-bit is perfectly fine to be effectively uniform selection.